scst-store/sca_scans/CVE.md

```diff
@@ -0,0 +1,42 @@
+# SCA Scanning Results
+
+## Scan Type:
+Software Composition Analysis scanning
+
+## Source of Scan:
+
+* Black Duck Binary Analysis (BDBA)
+* Grype
+
+## Version of Source:
+
+* BDBA version SDL 4.2
+* Grype version 0.25.1
+
+## CVEs
+### BDBA
+No vulnerabilities found in both the binaries of metadata-store and CLI.
```

Comment:
In the official docs, we're not using the term metadata-store, or even _The Store_. In this case it's easiest to call it API backend.

Reply:
Changed to "API backend"